packages/lodestar/test/utils/pyrmont.ts

```diff
@@ -3,28 +3,29 @@ import {createIBeaconConfig, IBeaconConfig} from "@chainsafe/lodestar-config";
 import {params} from "@chainsafe/lodestar-params/mainnet";
 import {goerliRpcUrl} from "../testParams";
 import {fromHexString} from "@chainsafe/ssz";
+import * as pyrmontTestnet from "@chainsafe/lodestar-cli/src/networks/pyrmont";
 
-export const medalla = {
+export const pyrmont = {
   providerUrl: goerliRpcUrl,
-  depositBlock: 3085928,
-  // Medalla optimized blocks for quick testing
+  depositBlock: pyrmontTestnet.depositContractDeployBlock,
+  // Pyrmont optimized blocks for quick testing
   blockWithDepositActivity: 3124889,
 };
 
 /**
- * Medalla specs
+ * Pyrmont specs
  */
-export function getMedallaConfig(): IBeaconConfig {
+export function getPyrmontConfig(): IBeaconConfig {
   const config = createIBeaconConfig(params);
   config.params.DEPOSIT_NETWORK_ID = 5;
-  config.params.DEPOSIT_CONTRACT_ADDRESS = Buffer.from("07b39F4fDE4A38bACe212b546dAc87C58DfE3fDC", "hex");
-  config.params.MIN_GENESIS_TIME = 1596546000;
-  config.params.GENESIS_DELAY = 172800;
-  config.params.GENESIS_FORK_VERSION = Buffer.from("00000001", "hex");
+  config.params.DEPOSIT_CONTRACT_ADDRESS = Buffer.from(pyrmontTestnet.beaconParams.DEPOSIT_CONTRACT_ADDRESS as string);
+  config.params.MIN_GENESIS_TIME = pyrmontTestnet.beaconParams.MIN_GENESIS_TIME as number;
+  config.params.GENESIS_DELAY = pyrmontTestnet.beaconParams.GENESIS_DELAY as number;
+  config.params.GENESIS_FORK_VERSION = Buffer.from(pyrmontTestnet.beaconParams.GENESIS_FORK_VERSION as string, "hex");
   return config;
 }
 
-export const goerliMedallaLogs = [
+export const goerliPyrmontLogs = [
```

Comment:
The data is from Medalla not Pyrmont. To prevent confusion you can change it to goerliTestnetDepositLog and add a JSDoc above clarifying that this is from the Medalla deposit contract 0x07b39F4fDE4A38bACe212b546dAc87C58DfE3fDC. Same for goerliPyrmontDepositEvents below

Reply:
hmm, yeah now i'm wondering if maybe i should revert this file to its original medalla.ts and just use that?  otherwise, we might have to update this every time the current testnet dies.  i think it's probably okay to use data from an old testnet since it's only testing based on the data structure and not necessarily trying to do live testnet testing.  how do you feel about that?

Reply:
Just call it something generic like sample testnet data and add a comment explaining that the data is taken from Medalla